packages/nuxt/src/module.ts

```diff
@@ -12,15 +15,37 @@ export interface ModuleOptions {
    * @see https://onyx.schwarz/development/#installation
    */
   disableGlobalStyles?: boolean;
+  /**
+   * This module will automatically detect if @nuxtjs/i18n is used within the project and configure itself to work with it.
+   * This group of options gives you the flexibility to fine tune this integration.
+   */
+  i18n?: {
+    /**
+     * Prefix to use for all translation keys comming from onyx. The modules translations can later be used under this prefix, e.g. `$t('onyx.optional')`.
+     * @default "onyx"
+     */
+    prefix?: string;
```

Comment:
Yes the propability is very low. Still it's basically no effort to just have the option so why not give people the option to change it in case they need to.